packages/editor/src/client/utility.ts

```diff
@@ -125,7 +125,12 @@ export function transformCssStringToObject(styleCustom: string): Record<string,
   return styleRules.reduce((previousValue: Record<string, unknown>, currentValue: string) => {
     const [key, value] = currentValue.split(':')
     if (key && value) {
-      return Object.assign(previousValue, {[key.trim()]: value.trim()})
+      return Object.assign(previousValue, {
+        [key
+          .toLowerCase()
+          .replace(/-(.)/gm, ($0, $1) => $1.toUpperCase())
+          .trim()]: value.trim()
+      })
```

Comment:
Please add a comment and describe what this function does. Nobody understands this.

Reply:
Its to extract the html properties of the pasted embed function and format as React. The function was already here, but needed to be adjusted to handle multi-word properties. `border-radius` -> `borderRadius`

Reply:
Will add comment.